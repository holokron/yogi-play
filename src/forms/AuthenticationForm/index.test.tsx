import * as React from 'react'
import * as renderer from 'react-test-renderer'
import AuthenticationForm from '.'

describe('@forms/AuthenticationForm', () => {
    it('renders correctly', () => {
        const tree = renderer.create(
            <AuthenticationForm 
                onSubmit={(emai: string, password: string): Promise<any> => new Promise(() => {})}
            />
        ).toJSON()
        
        expect(tree).toMatchSnapshot()
    })
})