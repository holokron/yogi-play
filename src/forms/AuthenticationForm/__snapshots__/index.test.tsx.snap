// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@forms/AuthenticationForm renders correctly 1`] = `
<form
  noValidate={true}
  onSubmit={[Function]}
>
  <div
    className="form-group"
  >
    <label>
      Email
    </label>
    <input
      className="form-control"
      name="email"
      onChange={[Function]}
      placeholder="Email"
      type="email"
      value=""
    />
  </div>
  <div
    className="form-group"
  >
    <label>
      Hasło
    </label>
    <input
      className="form-control"
      name="password"
      onChange={[Function]}
      placeholder="Hasło"
      type="password"
      value=""
    />
  </div>
  <button
    className="btn btn-primary"
    onClick={[Function]}
    type="submit"
  >
    Zaloguj
  </button>
</form>
`;
